src/pytorch_lightning/loggers/tensorboard.py

```diff
@@ -50,7 +50,7 @@
     from omegaconf import Container, OmegaConf
 
 
-class TensorBoardLogger(Logger):
+class TensorBoardLogger(Logger[SummaryWriter]):
```

Comment:
I don't think this will work because loggers are optional components so we defer their imports to the last possible moment (when they are actually used), whereas this needs to be available when the class is defined.